modules/system.js

```diff
@@ -245,6 +247,17 @@ System.prototype.nonceCompatible = function(nonce) {
 	return nonce && __private.nonce !== nonce;
 };
 
+/**
+ * Check if the ip exists in the peer blacklist present in the config file.
+ *
+ * @param ip
+ * @returns {boolean}
+ * @todo Add description for the params and the return value
+ */
+System.prototype.isBlacklisted = function(ip) {
+	return __private.blackListPeers.indexOf(ip) > -1;
+};
```

Comment:
Same method available to `peers` module, so its duplicate. Please keep it only in `peers` module.

Reply:
Yes we should have or or the other.